mythril/laser/ethereum/state/calldata.py

```diff
@@ -49,12 +63,16 @@ def __getitem__(self, item: Union[int, slice]) -> Any:
                 )
                 parts = []
                 while simplify(current_index != stop):
-                    parts.append(self._load(current_index))
+                    element = simplify(self._load(current_index))
```

Comment:
How about putting simplify() in the self._load() itself?

Reply:
Why?